### ISSUE TITLE
Generating launch configurations automatically

### ISSUE THREAD

**Issue by wleczny** [scalameta/metals-feature-requests#306]
### Is your feature request related to a problem? Please describe.

I would like Metals to generate launch configurations for the project.

### Describe the solution you'd like

Command (fe. `Generate launch configurations`), which will add configurations to the `launch.json` file based on build targets detected in the project.

Example of generated configurations:

```
{
  "type": "scala",
  "request": "attach",
  "name": "project",
  "buildTarget": "project_183d125c5c",
  "hostName": "localhost",
  "port": 5005
},
{
  "type": "scala",
  "request": "attach",
  "name": "project-test",
  "buildTarget": "project_183d125c5c-test",
  "hostName": "localhost",
  "port": 5005
}
``` 

### Describe alternatives you've considered

Adding configurations manually.

### Search terms

debugging, launch configuration

**Comment by kubukoz** [comment]
I see - the problem here is that scala-cli generates these arbitrary build target names, right? For users of sbt it's not as much of an issue, I think...

**Comment by tgodzik** [comment]
IT's still a bit of an issue since you need to find the target name. We should be able to try and figure out the target of the last focused file if none is specified.

**Comment by adpi2** [comment]
> IT's still a bit of an issue since you need to find the target name. We should be able to try and figure out the target of the last focused file if none is specified.

Would it be possible for Metals to ask the user to select the build target in a list of all build targets?

**Comment by tgodzik** [comment]
> > IT's still a bit of an issue since you need to find the target name. We should be able to try and figure out the target of the last focused file if none is specified.
> 
> Would it be possible for Metals to ask the user to select the build target in a list of all build targets?

Yeah, we can use the Metals Quick Pick for that, the same thing that is used for new Scala Project.
